app/helpers/admin/claims_helper.rb

```diff
@@ -50,5 +51,22 @@ def display_school(school)
       ].join(" ").html_safe
       sanitize(html, tags: %w[span a], attributes: %w[href class])
     end
+
+    def checking_deadline_warning(claim)
+      weekdays_until_checking_deadline = weekdays_between(Date.today, claim.checking_deadline_date)
+      return if weekdays_until_checking_deadline > 10
```

Comment:
Can we pull this `10` out into a constant so it's easier to follow the purpose of it (and see what it is)?

Reply:
Yeah, this value and the `2.weeks` in the scope are effectively the same thing, right?